openslides_backend/action/action_handler.py

```diff
@@ -65,7 +66,7 @@ class ActionHandler(BaseHandler):
     Action handler. It is the concrete implementation of Action interface.
     """
 
-    MAX_RETRY = 3
+    MAX_RETRY = 100
```

Comment:
This was probably for debugging I guess, please revert

Reply:
Reverted